include/wx/msw/gccpriv.h

```diff
@@ -77,9 +77,7 @@
         #define HAVE_W32API_H
     #endif
 #elif defined( __CYGWIN__ ) && !defined( HAVE_W32API_H )
-    #if ( __GNUC__ > 2 )
-        #define HAVE_W32API_H
-    #endif
+    #define HAVE_W32API_H
```

Comment:
Probably could get rid of this one entirely and just always include `<w32api.h>` here.

Reply:
I know nothing about how Cygwin or WINE, so I'd rather not touch that part.

Reply:
I'm going to remove it because not having to test for it in configure and define it in makefiles is a nice gain, so I'll just push the extra commits in this PR in a moment.